packages/block-editor/src/store/effects.js

```diff
@@ -219,6 +271,30 @@ export default {
 			)
 		);
 	},
+	MOVE_BLOCKS_DOWN: ( action, { getState } ) => {
+		const state = getState();
+		const settings = getSettings( state );
+
+		let direction = 'down';
+
+		if ( action.orientation === 'horizontal' ) {
+			direction = settings.isRTL ? 'left' : 'right';
+		}
+
+		announceBlockMoved( state, direction );
+	},
+	MOVE_BLOCKS_UP: ( action, { getState } ) => {
```

Comment:
It's odd to read an action type of "UP" while orientation changes it to right / left within. It'd be better to have a single "MOVE_BLOCKS" type and handle all cardinals within.

Reply:
This just uses the existing actions. Actually, there are no specific actions for "left" and "right". Not opposed to improvements but the current actions implementation is out of the scope of this issue.